### DISCUSSION TITLE
How to update from 1.21.1-dev to 1.22 latest using git?

### DISCUSSION THREAD

**Discussion by realex64** [FreshRSS/FreshRSS#5729] [Q&A] [answered]
Hi folks,

i am on 1.21.1-dev because it fixed an issue i had. Now 1.22 is out, i want to update to the new version using git.
I  used this:
```
cd /usr/share/freshrss
git reset --hard
git pull
cli/access-permissions.sh
chown -R www-data *
```
It seems to change a lot of files. But i end up with with 1.22.1-dev.  The next i tried was, I followed this https://freshrss.github.io/FreshRSS/en/admins/07_LinuxUpdate.html
At step 4 i changed from edge to latest. The result was:
```
git pull latest
fatal: 'latest' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
How can i update to stable 1.22, not "-dev"? 

Thank you very much!

Kind regards

Alex

**Comment by Frenzie** [comment]
`git switch latest`, possibly preceded by `git fetch`.

**Comment by Alkarex** [comment, marked as answer]
Hello,
You can track the `latest` branch https://github.com/FreshRSS/FreshRSS/tree/latest

Something like:

```sh
cd /usr/share/freshrss
git fetch --all --prune
git checkout -b latest origin/latest
```

**Reply by Tealk** [reply]
I also just switched back to latest, but the update still says that it wants to switch to edge:

![image](https://github.com/FreshRSS/FreshRSS/assets/12276250/2094b9fa-0281-45cc-a30e-6be88c59bd14)

**Reply by Alkarex** [reply]
That is another problem, due to the Web interface for the updates having some small glitches https://github.com/FreshRSS/FreshRSS/issues/5850
Trust what the git command line says.

**Comment by realex64** [comment]
I did follow Alkarex' suggestion and it worked. Thank you both for your quick reply...

Kind regards

Alex
